### ISSUE TITLE
upgrade rx_bloc_generator because it's not compatible with analyzer 5.2.0

### ISSUE THREAD

**Issue by SamerOrfali22** [Prime-Holding/rx_bloc#379]
upgrade rx_bloc_generator because it's not compatible with analyzer 5.2.0

**Comment by NiKoTron** [comment]
@SamerOrfali2 While PR under review, you can override dependency in your `pubspec.yaml` like this:

```yaml
dependency_overrides:
  rx_bloc_generator:
    git:
      url: git@github.com:Prime-Holding/rx_bloc.git
      path: packages/rx_bloc_generator
      ref: 04c12e6ea93d23655711dd747e6bbaa30f1a38b0
```

**Comment by StanevPrime** [comment]
This issue will be resolved in the next release - https://github.com/Prime-Holding/rx_bloc/blob/develop/packages/rx_bloc_generator/CHANGELOG.md

**Comment by StanevPrime** [comment]
The requested compatibility is now available in rx_bloc_generator: ^6.0.1
